apps/masterbots.ai/lib/hooks/useSonner.tsx

```
@@ -0,0 +1,71 @@
+              {config.title}
+            </h2>
+          </div>
+          <p className="text-sm dark:text-[#A1A1AA] text-[#71717A] pr-16">
+            {config.text}
+          </p>
+
+          <div className="absolute inset-y-0 right-3 flex items-center">
+            <button
+              className="border dark:border-[#27272A] border-[#E4E4E7] rounded-md py-1.5 px-2 text-sm dark:text-[#fafafa] text-[#09090B]"
+              onClick={() => toast.dismiss(t)}
+            >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)